src/DotLiquid.Tests/LiquidTypeAttributeTests.cs

```diff
@@ -43,6 +43,14 @@ public class MyLiquidTypeWithReservedKeyword
             public string Type { get; set; }
         }
 
+        [LiquidType("*")]
+        public class MyLiquidTypeWithConflictingGetter
+        {
+            public string Name { get; set; }
+
+            public string GetName() => Name;

```

Comment:
Isn't it that by having the GetName explicit method return the same Name value, we can't distinguish in the test if the response came from that method or from the generated Name property getter?

Reply:
True, I can adjust that to be unique. But previously it clashed and threw an error so this is already progress.